core/lib/dal/migrations/20230919154048_boojum-block-commitments.up.sql

```diff
@@ -1,7 +0,0 @@
--- Add up migration script here
-
-ALTER TABLE l1_batches
```

Comment:
Don't forget to remove entry for this migration from `_sqlx_migrations` on boojnet env. cc @ly0va